packages/react-components/theme-designer/package.json

```diff
@@ -20,13 +20,13 @@
     "@fluentui/react-components": "^9.61.5",
     "@fluentui/react-context-selector": "^9.1.75",
     "@fluentui/react-icons": "^2.0.245",
-    "@fluentui/react-storybook-addon-export-to-sandbox": "^0.1.0",
+    "@fluentui/react-storybook-addon-export-to-sandbox": "^0.0.0",
     "@fluentui/react-theme": "^9.1.24",
     "@fluentui/react-utilities": "^9.18.23",
     "@griffel/react": "^1.5.22",
     "@swc/helpers": "^0.5.1",
     "dedent": "^1.2.0",
-    "codesandbox-import-utils": "2.2.3"
+    "codesandbox-import-utils": "^2.2.3"
```

Comment:
wondering if this is still actual as we don't export to CS besides our addon configuration options ?